[ERROR] "vite-plugin-rewrite-all" resolved to an ESM file. ESM file cannot be loaded by `require`.

Steps to reproduce:

1) Clone prez-ui
2) ~> `yarn install`
3) ~> `yarn run build`

Output:
 [ERROR] "vite-plugin-rewrite-all" resolved to an ESM file. ESM file cannot be loaded by `require`.
 
See full error output here:  https://gist.github.com/ashleysommer/123e9186b948aa2fa24fb99fb72d880c


Also occurs with `yarn dev`

The error mentions to check out this page:
https://vitejs.dev/guide/troubleshooting.html#this-package-is-esm-only
But that doesn't help me much. I assume this used to work and recently broke, maybe a new incompatible version of vite-plugin-rewrite-all was released recently.

--- a/package.json
+++ b/package.json
@@ -25,7 +25,7 @@
         "floating-vue": "^2.0.0-beta.24",
         "n3": "^1.17.1",
         "pinia": "^2.1.7",
-        "vite-plugin-rewrite-all": "^1.0.1",
+        "vite-plugin-rewrite-all": "=1.0.1",
         "vue": "^3.3.0",
         "vue-router": "^4.2.5"
     },

